cluster/router/condition/router.go

```diff
@@ -149,6 +185,7 @@ func (c *ConditionRouter) Route(invokers []protocol.Invoker, url common.URL, inv
 		return result
 	} else if c.Force {
 		rule, _ := url.GetParamAndDecoded(constant.RULE_KEY)
+		localIP, _ := gxnet.GetLocalIP()
 		logger.Warnf("The route result is empty and force execute. consumer: %s, service: %s, router: %s", localIP, url.Service(), rule)
 		return result
```

Comment:
`return invoker`? The `c.Force` is true but return `result` which is empty, it doesn't make sense.

Reply:
I am not think so base on this scene , it must ```return result```, becauce it must the same to Dubbo , otherwise ```force``` key in config file will be different means.

I think ``` else if c.Force ``` change to ``` else if !c.Force ``` will better